Documentation/README_Solver.md

```diff
@@ -49,6 +49,17 @@ The *Tracked Target Type* property of the [`SolverHandler`](xref:Microsoft.Mixed
 ![Solver](Images/Solver/TrackedObjectType-Example.gif)  
 *Example of various properties associated with each TrackedTargetType*
 
+> [!IMPORTANT]
+> Most Solvers use the forward vector of the tracked transform target supplied by the `SolverHandler`. When using a *Hand Joint* Tracked Target Type, the forward vector of the palm joint may point through the fingers and not through the palm. This depends on the platform supplying the hand joint data. For input simulation and Windows Mixed Reality, it is the *up vector* that points up through the palm (i.e green vector is up, blue vector is forward).
+> 
+> ![Solver](Images/Solver/HandJoint_ForwardUpVectors.png)
+> 
+> To overcome this, update the *Additional Rotation* property on the `SolverHandler` to **<90, 0, 0>**. This will ensure the forward vector supplied to Solvers is pointing through the palm and outward away from the hand.
```

Comment:
```suggestion
> To overcome this, update the *Additional Rotation* property on the `SolverHandler` to **<90, 0, 0>**. This will ensure the forward vector supplied to solvers is pointing through the palm and outward away from the hand.
```